assets/subworkflows/espresso/processing_find_minima.yml

```diff
@@ -0,0 +1,23 @@
+application:
+    name: python
+    version: 3.8.6
+method:
+    name: UnknownMethod
+model:
+    name: UnknownModel
+name: Find ESP Value
+units:
+-   config:
+        name: Find Extrema
+        execName: python
+        flavorName: generic:processing:find_extrema:scipy
+        flowchartId: python-find-extrema
+    type: executionBuilder
```

Comment:
Maybe let's just use executionBuilder? Or otherwise, use assignmentBuilder, etc. when we have additional config?

Reply:
`type: executionBuilder` really only comes into effect in `createUnit` (subworkflow/create.js), where it is used to create the config for the execution unit before calling `unitFactoryCls.create()`. I don't quite understand why this was done, but my guess is that the config for the execution unit is somewhat involved and generating the config via `ExecutionUnitConfigBuilder` helps to keep the YAML less verbose. By the way, all of the execution units in the YAML assets are defined this way.